geomesa-hbase/pom.xml

```diff
@@ -32,6 +32,11 @@
                 <artifactId>geomesa-hbase-datastore_${scala.binary.version}</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.google.guava</groupId>
+                <artifactId>guava</artifactId>
+                <version>12.0.1</version>
```

Comment:
Yeah, I had that but reverted it, not sure what we wan't to do here. I don't know how to tell if it's effecting bigtable stuff.